boskos/janitor/janitor.py

```diff
@@ -53,6 +53,7 @@
     Resource('', 'compute', 'networks', 'subnets', 'region', None, True, True),
     Resource('', 'compute', 'networks', None, '', None, False, True),
     Resource('', 'compute', 'routes', None, None, None, False, True),
+    Resource('', 'compute', 'tpus', None, 'zone', None, False, True),
```

Comment:
looks like you can flip `tolerate` to True to skip errors

Reply:
(but that still dumps out the error which will pollute the logging :-\ maybe we want to check for the api first?)